subsys/event_manager/event_manager.c

```diff
@@ -59,11 +61,20 @@ static void event_processor_fn(struct k_work *work)
 
 		trace_event_execution(eh, true);
 		if (IS_ENABLED(CONFIG_DESKTOP_EVENT_MANAGER_SHOW_EVENTS)) {
-			printk("e: %s ", et->name);
-			if (et->print_event) {
-				et->print_event(eh);
+			if (et->log_event) {
+				char event_log_buf[CONFIG_DESKTOP_EVENT_MANAGER_EVENT_LOG_BUF_LEN];
+				size_t pos;
+
+				pos = et->log_event(eh, event_log_buf,
+					CONFIG_DESKTOP_EVENT_MANAGER_EVENT_LOG_BUF_LEN);
+				__ASSERT_NO_MSG(
```

Comment:
please do not use asserts in code that can be enabled in the release mode
btw - in this case it seems we can ignore error checking (the only visible problem will be truncated string)